src/server/api/bannerRouter.ts

```diff
@@ -105,6 +110,17 @@ export const buildBannerRouter = (
                 requiredRegion,
                 targetingMvtId,
             );
+            const isVatCompliantCountry = variantAmounts?.testName !== 'VAT_COMPLIANCE';
+
+            const choiceCardsSettings =
+                design?.visual?.kind === 'ChoiceCards' && isVatCompliantCountry
+                    ? getChoiceCardsSettings(
+                          requiredRegion,
+                          'Epic',
```

Comment:
Is this Epic? Or Banner1 ?

Reply:
Good spot!

Reply:
I've pushed a commit to use the correct channel name